generic/proof-shell.el

```diff
@@ -1063,6 +1079,33 @@ being processed."
       ;; nothing to do: maybe we completed a list of comments without sending them
 	(proof-detach-queue)))))
 
+(defun start-prover-with-priority-items-maybe ()
```

Comment:
Nitpick: shouldn't this defun identifier start with a `proof-` prefix, just to avoid a potential global namespace clash?